docs/AboutUs.md

```diff
@@ -9,9 +9,9 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Ravi Ananya
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/banana3021.png" width="200px">
```

Comment:
Have you uploaded your own picture? Otherwise, the website will not shows your own picture.